builtin/stash--helper.c

```diff
@@ -18,6 +18,7 @@ static const char * const git_stash_helper_usage[] = {
 	N_("git stash--helper ( pop | apply ) [--index] [-q|--quiet] [<stash>]"),
 	N_("git stash--helper branch <branchname> [<stash>]"),
 	N_("git stash--helper clear"),
+	N_("git stash--helper store [-m|--message <message>] [-q|--quiet] <commit>"),
```

Comment:
Please mention in the commit message that the usage was forgotten in the shell script version (which I think it was, but it *might* be missing on purpose... but then, I can't find it in the commit log), so this is more than a literal conversion, but in a good sense.